modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -102,7 +102,22 @@ void gd_mono_profiler_init() {
 	bool profiler_enabled = GLOBAL_DEF("mono/profiler/enabled", false);
 	if (profiler_enabled) {
 		mono_profiler_load(profiler_args.utf8());
+		return;
 	}
+
+	char* mono_env_ops;
+	size_t len;
+	_dupenv_s(&mono_env_ops, &len, "MONO_ENV_OPTIONS");
```

Comment:
I think `_dupenv_s` is windows only. You can use `OS::get_singleton()->get_environment("MONO_ENV_OPTIONS")` instead.